test/common/schema/definitions.spec.js

```diff
@@ -0,0 +1,62 @@
+const ajv = require('ajv');
+const applicationSchema = require('../../../src/common/schema/application');
+const chai = require('chai');
+const expect = chai.expect;
+
+function definitionValidator(field) {
+  const tinySchema = {
+    $schema: 'http://json-schema.org/draft-04/schema#',
+    type: 'object',
+    properties: {
+      field: applicationSchema.definitions[field]
+    }
+  };
+
+  return function validator(value) {
+    const fn = ajv({ allErrors: true, errorDataPath: 'property', removeAdditional: 'failing', useDefaults: true }).compile(tinySchema);
```

Comment:
Can we yank the ajv({....}) out into a utility function in some file shared between test and prod?  I think ajv is used in 3 places now and they should all really be invoked with exactly the same parameters.